docs/contributing/common-scenarios.html

```diff
@@ -49,44 +49,46 @@ <h3 id='new-polyfill'>You want to add support for a feature not currently in the
-			<li>Save your polyfill code in a file called <code>polyfill.js</code> in a new subdirectory of the <code>/polyfills</code> directory, named for the JavaScript global that accesses the feature (or if the feature has no JavaScript API, use a descriptive name prefixed with a tilde, e.g. <code>~html5-elements</code>)</li>
-			<li>Add a well defined config.json, following the guidelines for <a href='/v{{apiversion}}/docs/contributing/authoring-polyfills#configuration'>configuring polyfills</a>.</li>
+			<li>If you want to use a third party polyfill available on npm:<ol>
+				<li>Write a well defined config.json, following the guidelines for <a href='/v{{apiversion}}/docs/contributing/authoring-polyfills#configuration'>configuring polyfills</a>, and including an <code>install</code> section refering to the source module and path.</li>
+				<li>Add the npm module to our package.json file and regenerate the shrinkwrap.</li>
+			</ol></li>
+			<li>If you want to write an original polyfill:<ol>
+				<li>Follow our guidelines for the best practice <a href='/v{{apiversion}}/docs/contributing/authoring-polyfills#authoring-guidelines'>architecture</a> of original polyfills</li>
+				<li>Save your polyfill code in a file called <code>polyfill.js</code> in a new subdirectory of the <code>/polyfills</code> directory, named for the JavaScript global that accesses the feature (or if the feature has no JavaScript API, use a descriptive name prefixed with a tilde, e.g. <code>~html5-elements</code>)</li>
+				<li>Add a well defined config.json, following the guidelines for <a href='/v{{apiversion}}/docs/contributing/authoring-polyfills#configuration'>configuring polyfills</a>.</li>
+				<li>Write a reasonably comprehensive set of tests in a <code>tests.js</code> file, and check that they pass in all the browsers you have targeted.</li>
+			</ol></li>
 			<li>Add a feature detect in a <code>detect.js</code> file if it's possible to detect the feature via a JavaScript expression.</li>
-			<li>Write a reasonably comprehensive set of tests in a <code>tests.js</code> file, and check that they pass in all the browsers you have targeted.</li>
 			<li>Run <code>grunt ci</code> to check that library still passes our continuous integration tests. For help connecting to a Sauce Labs account, see <a href='/v{{apiversion}}/docs/contributing/testing#runing-tests-sauce'>running tests on Sauce Labs</a>.</li>
-			<li>Open a pull request, including links to evidence of agreement on browser support from reliable sources, ie MDN, caniuse and/or Kangax, a screenshot of the test results in a targeted browser, and the results of your CI test run.</li>
-			<li>State your agreement to our contribution terms.</li>
+			<li>Open a pull request, including links to evidence of browser support from reliable sources, ie MDN, caniuse and/or Kangax, a screenshot of the test results in a targeted browser, and the results of your CI test run.</li>
```

Comment:
The results of the CI test run will always be a failure as the pull request will be based from a forked git repository which does not have access to our secret environment variables. Perhaps we should write something which explains that the CI tests will fail in pull requests made by forks.